shards/input/detached.hpp

```diff
@@ -116,7 +114,7 @@ struct DetachedInput {
     swapBuffers();
 
     synthesizeKeyUpEvents(inputState);
-    synthesizeKeyDownEvents(inputState);
+    // synthesizeKeyDownEvents(inputState);
```

Comment:
Hmm? 😅

Reply:
I'll clean up/remove the comments.
Thing is here that some events come from SDL directly and some I manually detect just in case SDL drops a key/button up event